db/redis.go

```diff
@@ -0,0 +1,42 @@
+package db
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/gomodule/redigo/redis"
+)
+
+var RedisPool *redis.Pool
+
+func GenerateRedisPool(address string) {
+	pool := &redis.Pool{
+		MaxActive: 5,
+		MaxIdle:   5,
+		Wait:      true,
+		Dial: func() (redis.Conn, error) {
+			return redis.Dial("tcp", address)
```

Comment:
Fixed in 8dddb821312aa28190dac06d1966b94832643d76

Reply:
The workflow failed remind me about why we cannot use environment variable in the test. Because we ignore all the .env file except .env.example so in the CI all the tests that need env variable will fail.

Is it acceptable to push `.env.test` to git repo?

Reply:
Yes, it's okay to commit the file. Only the production one should not be commited.

Reply:
Add `.env.debug` and `.env.test` in 26621e16d3203e0772e4b832c5da17b50236e1ce

Reply:
Think of updating the README files as well 

![image](https://user-images.githubusercontent.com/696529/102455250-d5f4bb80-4071-11eb-8b0f-82e9c2a4ec89.png)

Reply:
Updated in 47a30a7178417b517806e5610cbf85cedca4b95b